test/extended/operators/cluster.go

```diff
@@ -0,0 +1,251 @@
+package operators
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/beorn7/perks/quantile"
+
+	"github.com/beorn7/perks/histogram"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	coreclient "k8s.io/client-go/kubernetes/typed/core/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[Feature:Platform][Suite:openshift/smoke-4] Managed cluster should", func() {
+	defer g.GinkgoRecover()
+
+	g.It("have no crashlooping pods in core namespaces over two minutes", func() {
```

Comment:
this also runs during bootstrapping or do we wait for stabilization? With our "eventual running pods" approach 2 min does not sound much.

Reply:
It’s post bootstrapping.  A bootstrapping cluster isn’t conformant and I think we can safely gate on that.